ice.c

```diff
@@ -1406,6 +1416,13 @@ gint janus_ice_handle_destroy(void *core_session, janus_ice_handle *handle) {
 		return JANUS_ERROR_HANDLE_NOT_FOUND;
 	}
 	janus_mutex_unlock(&plugin_sessions_mutex);
+	janus_mutex_lock(&event_loops_mutex);
+	if(handle->static_event_loop != NULL) {
+		janus_ice_static_event_loop *loop = (janus_ice_static_event_loop *)handle->static_event_loop;
+		loop->handles--;
```

Comment:
I prefer keeping the mutex, to avoid the number of handles loops keep track of change while we're choosing a candidate, but let's wait for more feedback.